package.json

```diff
@@ -170,7 +170,7 @@
     "commitmsg": "node scripts/check-commit-message.js",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint --ext md --ext ts --ignore-pattern dist .",
-    "lint:md": "markdownlint ./.github docs",
```

Comment:
I've remove `.github` for now since I don't think we should have things such as `<!-- markdownlint-disable MDXXX ... -->` in files as the `ISSUE_TEMPLATE.md` as that may confuse users.